src/components/toast-notifications/toast-notifications.html

```diff
@@ -1,5 +1,7 @@
 <div class="toast-notifications-list-pf">
-  <div ng-repeat="(notificationID, notification) in notifications track by notification.trackByID" ng-if="!notification.hidden || notification.isHover"
+  <div
```

Comment:
Just added `notification.skipToast` to the `ng-if` here.

Reply:
Work for everyone?  If we good, this is prob ready to roll.

Reply:
Might be better to just not add the notification here:

https://github.com/openshift/origin-web-common/blob/master/src/components/toast-notifications/toastNotifications.js#L59

```js
if (notification.skipToast) {
  return;
}
$scope.$evalAsync(...)
```

Reply:
Great, updated.  @dtaylor113 you can eliminate that line if you are working off of this PR.